### PR TITLE
Prac 5 code review request 1

### PR DESCRIPTION
**Pull request by jc740631**
please review my code \@cue-aus

### CODE REVIEW

File: week 4/week 5/prac/state_name_1.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT" : "Northern Territory", "WA" : "Western Australia", "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
```
Comment (cue-aus) [inline week 4/week 5/prac/state_name_1.py]: remove space at NT and WA

File: week 4/week 5/prac/state_name_1.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+print(CODE_TO_NAME)
+
+state_code = input("Enter short state: ")
+while state_code != "":
+    if state_code in CODE_TO_NAME:
+        print(state_code, "is", CODE_TO_NAME[state_code])
+    else:
+        print("Invalid short state")
+    state_code = input("Enter short state: ")
```
Comment (cue-aus) [inline week 4/week 5/prac/state_name_1.py]: add one blank line at the end

**Review by cue-aus — APPROVED** [review]
Nothing more :)

**Comment by jc740631** [comment]
Ok  thanks Cue

**Pull request closed without merging**